simple-calculator/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
```

Comment:
Update this file according to the html/css style guide, also try to separate css from html if possible please.